proxies/live/apiproxy/policies/ServiceCallout.CallHealthcheckEndpoint.xml

```diff
@@ -13,7 +13,7 @@
         <LoadBalancer>
             <Server name="immunisation-history"/>
         </LoadBalancer>
-        <Path>/_status</Path>
+        <Path>/immunisation-history/_status</Path>
```

Comment:
Is there a clear reason to do this in this repository vs the dps-dda repository?

Reply:
I thought the old path was incorrect to begin with.
for exemptions api, we also use /covid-medical-exemption/_status as status endpoint

Reply:
Agreed. CME isn't linked up at the moment so it's not a huge issue.